packages/volto/news/5645.bugfix

```diff
@@ -0,0 +1 @@
+Fixes Toolbar, More, Add and Outside Click Handling function Bug @singhcoder694
```

Comment:
This is a vague and useless news item. See https://6.docs.plone.org/contributing/index.html#change-log-entry

The format should be:

Changed [a thing] from [something] to [another something]. @singhcoder694

Reply:
Hello @singhcoder694 , I have already raised a PR for this issue soon after filing it.

Reply:
Ohhkkk.... u may proceed🙂